Tests/ML/utils/test_model_util.py

```diff
@@ -43,10 +43,6 @@ def create_model_and_store_checkpoint(config: ModelConfigBase, checkpoint_path:
     if machine_has_gpu:
         model = model.cuda()  # type: ignore
     trainer.model = model
-    # Before saving, the values for epoch and step are incremented. Save them here in such a way that we can assert
```

Comment:
why deleted?

Reply:
This is no longer necessary for the same reason as the other change of epoch value: the epochs are now incremented at a different time and these lines now break a number of tests.